.github/workflows/CI.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3.3.0
```

Comment:
As per Github recommendations for using 3rd-party actions, we should specify a concrete tag version number.